wandb/cli/cli.py

```diff
@@ -1214,7 +1214,7 @@ def agent(ctx, project, entity, count, sweep_id):
     #                'args': ['--max_epochs=10']})
 
 
-@cli.command(context_settings=CONTEXT, help="Run the W&B local sweep controller")
+@cli.command(context_settings=CONTEXT, help="Run the W&B server sweep controller")
```

Comment:
is this relevant? It is just the sweeps controller...

Reply:
good catch, this is not relevant, it is a different thing.

Reply:
er.. this is still called local sweep controller.